docs/index/index-management.md

```diff
@@ -73,69 +71,69 @@ If user perform following command on the main table, system will return failure.
     not, the operation is allowed, otherwise operation will be rejected by throwing exception.
 3. Partition management command: `ALTER TABLE ADD/DROP PARTITION`.
 
-If user do want to perform above operations on the main table, user can first drop the datamap, perform the operation, and re-create the datamap again.
+If user do want to perform above operations on the main table, user can first drop the index, perform the operation, and re-create the index again.
 
-If user drop the main table, the datamap will be dropped immediately too.
+If user drop the main table, the index will be dropped immediately too.
 
-We do recommend you to use this management for index datamap.
+We do recommend you to use this management for index.
 
 ### Manual Refresh
 
-When user creates a datamap specifying manual refresh semantic, the datamap is created with status *disabled* and query will NOT use this datamap until user can issue REBUILD DATAMAP command to build the datamap. For every REBUILD DATAMAP command, system will trigger a full rebuild of the datamap. After rebuild is done, system will change datamap status to *enabled*, so that it can be used in query rewrite.
+When user creates a index specifying manual refresh semantic, the index is created with status *disabled* and query will NOT use this index until user can issue REFRESH INDEX command to build the index. For every REFRESH INDEX command, system will trigger a full rebuild of the index. After rebuild is done, system will change index status to *enabled*, so that it can be used in query rewrite.
 
-For every new data loading, data update, delete, the related datamap will be made *disabled*,
-which means that the following queries will not benefit from the datamap before it becomes *enabled* again.
+For every new data loading, data update, delete, the related index will be made *disabled*,
+which means that the following queries will not benefit from the index before it becomes *enabled* again.
 
-If the main table is dropped by user, the related datamap will be dropped immediately.
+If the main table is dropped by user, the related index will be dropped immediately.
 
 **Note**:
-+ If you are creating a datamap on external table, you need to do manual management of the datamap.
-+ For index datamap such as BloomFilter datamap, there is no need to do manual refresh.
++ If you are creating a index on external table, you need to do manual management of the index.
++ For index such as BloomFilter index, there is no need to do manual refresh.
  By default it is automatic refresh,
- which means its data will get refreshed immediately after the datamap is created or the main table is loaded.
- Manual refresh on this datamap will has no impact.
+ which means its data will get refreshed immediately after the index is created or the main table is loaded.
+ Manual refresh on this index will has no impact.
 
 
 
-## DataMap Catalog
+## Index Catalog
```

Comment:
Please remove this section, as all indexes will be stored in carbontable itself in tableproperties